amundsen_application/static/js/utils/numberUtils.ts

```diff
@@ -0,0 +1,9 @@
+import { getNumberFormat } from 'config/config-utils';
+
+export function formatNumber(value) {
+  const numberformat = getNumberFormat();
+  if (numberformat) {
+    return Intl.NumberFormat(numberformat).format(value);
+  }
+  return Intl.NumberFormat().format(value);
+}
```

Comment:
I think you could do:
```suggestion
export function formatNumber(value) {
  const numberformat = getNumberFormat() || undefined;

    return Intl.NumberFormat().format(value);
}
```